OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -196,6 +196,9 @@ public void OnAllRemovedFromWorld(Actor[] actors, LuaFunction func)
 			{
 				try
 				{
+					if (!group.Contains(m))
+						return;
```

Comment:
Remove will also tell you if the element was in the collection, i.e you could do:
```csharp
if (!group.Remove(m))
    return;
```